examples/hackernews/reactive_service/server.js

```diff
@@ -1,15 +1,5 @@
-import postgres from "postgres";
 import { runService } from "@skipruntime/server";
-import { serviceWithInitialData } from "./dist/hackernews.service.js";
+import { initService } from "@skipruntime/wasm";
```

Comment:
We shouldn't need to import from internal packages in examples. Looking around, I think that the package reorg left runService uncallable using only public apis, so probably we should export initService from core/src/api.ts before merging this PR. I don't know whether to do that before or after https://github.com/SkipLabs/skip/pull/686.

Reply:
It is publicly callable via @skipruntime/server though -- that's what's happening here, and the example works resolving everything through the NPM registry

Reply:
Maybe I'm confused but I don't see ServiceInstance or initService exported from @skipruntime/server?

Reply:
(I'm not saying that this code doesn't actually work, just that it is pointing out that people need to use things from internal packages, that are also not documented.)

Reply:
Ahh, I see what you mean -- I was referring to `runService` not `initService` as being callable via `@skipruntime/server`

As it stands, user code needs to call the`initService` from either `@skipruntime/wasm` or `@skipruntime/native`.  I'm not sure this is a bad thing though (outside of the question of documentation) since they initialize the service differently but are both exported publicly.  Under the current dependency structure, it would be difficult to expose `initService` at the level of `@skipruntime/core`

(`ServiceInstance` is already exported in `@skipruntime/core` so that's fine)

Reply:
So as it stands, user code needs to change depending on whether the wasm or native version is desired? I didn't appreciate that. Ok, this PR is not making anything worse. I'm not sure yet what to do about this situation though.

Reply:
I've got a draft PR baking adding the two `initService`s to the docusaurus site, but there's a bit of a question of what to do with `ServiceInstance` -- we don't want its actual structure to be part of the public API, but it is needed as an opaque/existential type, i.e. the thing produced by `initService` and passed to `runService`.

I've set that aside to work on other things for now, but we should chat about how to make things better :)